scripts/dist.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/python3
+
+import os
+from pathlib import Path
+import shutil
+import subprocess
+
+base = Path(os.getenv('MESON_DIST_ROOT'))
+
+base.joinpath('.gitattributes').unlink()
+shutil.rmtree(base / '.github')
+shutil.rmtree(base / 'misc')
+shutil.rmtree(base / 'test/cases')
```

Comment:
You can use gitattributes export-ignore here, but Meson only makes use of that starting with Meson 0.60...

Reply:
considering that we're already constraining the environment where a source distributions is built with a specific version of doxygen it might just be worth the extra simplification to also assume meson >=0.60.

@bgilbert either way, the existing gitattributes is really autoconf specific, so either it can be removed completely as part of this patch, or should be updated to add export-ignore for those paths that should not be part of the source distribution.

Reply:
Switched to gitattributes, and added a release checklist item to confirm Meson >= 0.60.